config/projects.yaml

```diff
@@ -360,8 +358,6 @@ form-extensions:
                 symfony/symfony: ['4.4.*', '5.3.*', '5.4.*', '6.0.*']
 
 google-authenticator:
-    excluded_files:
-        - README.md
```

Comment:
this project has no docs, and I think it does have some information in the readme

Reply:
indeed